src/goto-harness/goto_harness_parse_options.cpp

```diff
@@ -50,6 +84,7 @@ int goto_harness_parse_optionst::doit()
 
   auto factory_options = collect_generate_factory_options();
 
+  register_language(new_ansi_c_language);
```

Comment:
:no_entry_sign: Could a why comment explaining why this is needed be added here

Reply:
@thk123 This snuck in from debugging earlier, it's not needed at all.